docs/blog/newsletter-june-2020.rst

```diff
@@ -0,0 +1,132 @@
+
+And if you’re in a time zone closer to UTC like me, keep your eyes peeled - we’ll be announcing dates for virtual Prague sometime soon.
+
+With that, let’s see what the community has been talking about this month.
+
+
+---------------------------------
+Broken links and how to find them
+---------------------------------
+
+If your product includes links to docs topics, you're likely quite aware of those links. Perhaps you’re careful to fix the product links when docs URLs change, or maybe you just avoid changing link anchors in the docs altogether when you revise or update.
+
+But it gets a bit more complicated when more contributors are making docs updates, and a recent Slack discussion included a helpful idea for this scenario (as long as your docs pages are output in HTML).
+
+First, create a docs page that lists only the product links to docs topics. Then, create a separate testing environment for your docs site that includes the product links page in the HTML output. Finally, before you publish docs updates, build your pages in the testing environment and use a tool like `html-proofer <https://github.com/gjtorikian/html-proofer>`__ to find any broken product links.
+
+
+---------------------------------
+Types of docs and how to use them
+---------------------------------
+
+Back in 2017, Daniele Procida gave an often cited talk, `The four kinds of documentation, and why you need to understand what they are </videos/eu/2017/the-four-kinds-of-documentation-and-why-you-need-to-understand-what-they-are-daniele-procida/>`__, written up into `this article <https://documentation.divio.com/>`__. Our community spent some time discussing the details of how to apply Danielel's type distinctions, with some help from Daniele himself. As a refresher, the four types are:
+
+* Tutorials, or learning-oriented docs 
+* How-to guides, or problem-solving docs
+* Explanations, or understanding-oriented docs
+* Reference, or information-oriented docs
+
+Here's how the conversation built on these types:
+
+* Regardless of the type of doc, make sure to provide context to orient the user. Do they need to learn more before they read? What level of expertise is the doc aimed at?
+* Other terms used to refer to the same types are "concept" (for explanations), and "procedure" -- which could cover both how-tos and tutorials, because all share a step-by-step approach. But a more nuanced distinction emerged. Tutorials can be thought of as step by step how-tos, but involving larger tasks or a broader grasp of how things work. How-to docs are smaller, and show how to accomplish only a single task.
```

Comment:
I would normally hyphenate "step-by-step" even positioned next to "how-tos" as it is here